scripts/binary-tarball.nix

```diff
@@ -23,7 +23,7 @@ in
 runCommand "nix-binary-tarball-${version}" env ''
   cp ${installerClosureInfo}/registration $TMPDIR/reginfo
   cp ${./create-darwin-volume.sh} $TMPDIR/create-darwin-volume.sh
-  substitute ${./install-nix-from-closure.sh} $TMPDIR/install \
+  substitute ${./install-nix-from-tarball.sh.in} $TMPDIR/install \
```

Comment:
I am ok with the name `tarball.sh` but the `.in` extension is confusing for other reasons. `.in` is the extension used by autotools/meson to do string substitution. However this file is not substituted by `autotools` so it would be confusing by the reader if it was. We also have this extension for any of the scripts belwo.

Reply:
I thought that double `.in` extension is generic for scripts that are not ready to be used without substitution. It is surprising to know that `.in` extension is an embedded feature of `autotools` and that `autotools` is still alive in the project like `nix`. :D

I will strip the `.in`. Just curious - does it break the build, because `autotools` starts to process it?

Reply:
It won't start to process it. But people that read see the filename might believe it's part of the files to be processed.

Reply:
@Mic92 this file needs to be processed, because it needs these variables to be substituted.
```
nix="@nix@"
cacert="@cacert@"
```
It won't work if I just copy new version from the repo over `/install`.

Reply:
@Mic92 got back to `.sh` extension.

Reply:
Thanks.